subsys/bluetooth/controller/hci/hci.c

```diff
@@ -8380,7 +8449,13 @@ static void encode_control(struct node_rx_pdu *node_rx,
 
 #if defined(CONFIG_BT_CTLR_SCAN_REQ_NOTIFY)
 	case NODE_RX_TYPE_SCAN_REQ:
+#if defined(CONFIG_BT_EXT_ADV)
```

Comment:
```suggestion
#if defined(CONFIG_BT_CTLR_ADV_EXT)
```

Reply:
Thanks, done. However I cannot find where are the tests with BSIM failing, I don't see logs.

Reply:
@giansta I think the server pinged out or something (happened to one of my PRs as well). I restarted the test :)

Reply:
Thanks, it was resolved later :-)